### PR TITLE
feat: added screens

### PR DESCRIPTION
**Pull request by Chakravarthi0**
added the following screens:
- sign-in screen
- sign-up screen
- home screen
- explore screen
- bookmarks screen
- profile screen

### CODE REVIEW

**Review by BelideNikhil — APPROVED** [review]
The UI looks great.
Dark mode is litt
The components are well named and well structured as well
God to merge!

File: src/components/comment/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
```
Comment (msdeshmukh009) [inline src/components/comment/Comment.jsx]: You can remove this import as it is not necessary.

File: src/layouts/sidebar/SideBar.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        </li>
+        {/* <li>
+          <Link
+            to={"/profile"}
+            className="flex items-center p-2 font-normal text-slate-900 rounded-lg dark:text-white hover:bg-slate-200 dark:hover:bg-slate-700 text-3xl"
+          >
+            <MdOutlinePersonOutline />
+            <p className="ml-3 text-xl hidden xl:block">Profile</p>
+          </Link>
+        </li> */}
```
Comment (msdeshmukh009) [inline src/layouts/sidebar/SideBar.jsx]: You can remove this comment if not required.

File: src/pages/signin/SignIn.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { SigninCard } from "../../components";
+
+const SignIn = () => {
+  return (
+    <div>
+      <SigninCard />
+    </div>
```
Comment (msdeshmukh009) [inline src/pages/signin/SignIn.jsx]: Any particular reason for wrapping this component in div?

**Review by msdeshmukh009 — APPROVED** [review]
The rest of the code looks good.
UI is 🔥.
Nice work.

**Comment by Chakravarthi0** [comment]
> The rest of the code looks good. UI is fire. Nice work.

Thanks for reviewing, I have made the suggested changes

**Pull request merged**